### PR TITLE
Talk submission from My talks

### LINKED ISSUE #157: [speaker] submit a talk from the talk page
It'd be nice to be able to submit a talk from the talk page.

It would require to add a "Submit" button in the "Submissions" card of the talk :

image

Then slightly change the submission process to replace the "Talk selection" step by an "Event selection" step :

image

This is just a suggestion, what do you think ?

### PR DESCRIPTION
**Pull request by nlepage**
fix #157

### CODE REVIEW

File: src/screens/speaker/talk/submission/submission.container.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Submission from './submission'
+
+const mapStore = (store) => {
+  const talkId = getRouterParam('talkId')(store.getState())
+  const talk = store.ui.speaker.myTalks.get(talkId) || {}
+  return {
+    loaded: !!talk,
+    talkId: talk.id,
+    talkTitle: talk.title,
+    load: () => store.dispatch('@@ui/ON_LOAD_SPEAKER_TALKS'),
```
Comment (bpetetot) [inline src/screens/speaker/talk/submission/submission.container.js]: I think you only need to load the talk defined in the route with `@@ui/ON_LOAD_TALK` action.
```js
store.dispatch('@@ui/ON_LOAD_TALK')
```

File: src/store/listeners/reactions/events.js [hunk 2]
```diff
@@ -48,3 +48,13 @@ export const fetchOrganizerEvents = reaction(async (action, store) => {
   store.ui.organizer.myEvents.reset()
   store.ui.organizer.myEvents.set(events)
 })
+
+export const fetchSpeakerEvents = reaction(async (action, store) => {
+  const result = await fetchEvents()
+  const events = result.docs.map(ref => ({ id: ref.id, ...ref.data() }))
+  // set events in the store
+  store.data.allEvents.set(events)
```
Comment (bpetetot) [inline src/store/listeners/reactions/events.js]: You can use `store.data.events` that is done to store all events data

File: src/store/reducers/data/index.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { keyValue } from 'k-ramel'
 
 export default {
-  users: keyValue({ key: 'uid' }),
+  allEvents: keyValue({ key: 'id' }),
   events: keyValue({ key: 'id' }),
```
Comment (bpetetot) [inline src/store/reducers/data/index.js]: You don't need to add `allEvents`, you can reuse `events`

**Review by bpetetot — DISMISSED** [review]
Great work Nico ! Thanks 👍 
I have little remarks... but I have tested it and it's really useful in the submission process.

File: src/store/listeners/reactions/events.js (hunk 2 above)
Comment (nlepage) [reply src/store/listeners/reactions/events.js]: OK, I wasn't sure.
`store.data.events` is already used by `fetchOrganizerEvents` and contains only the events of one organizer.
That's why I created `store.data.allEvents`.
Are you sure it's OK to use `store.data.events` anyway ?

Comment (nlepage) [reply src/store/listeners/reactions/events.js]: OK I made a test, and it's not problem, I'll add a commit.

**Review by bpetetot — APPROVED** [review]
👌 Great ! Thanks a lot

**Pull request merged**